Makefile

```diff
@@ -16,6 +16,11 @@ ifeq ($(OS),$(filter $(OS), FreeBSD OpenBSD NetBSD))
 CADDFLAG += -lexecinfo
 endif
 
+ifeq ($(OS),$(filter $(OS),  Linux))
+CADDFLAG += -lm
+$(info CADDFLAG is $(CADDFLAG))
```

Comment:
Do we need this here?

Reply:
No, that's a mistake. I'll fix it and make a new pull request.

Reply:
You needn’t! You can just add the commit, the PR will be updated :)

Reply:
I've updated the code. I moved the `-l m` flag into *`LUA_FLAGS`* variable instead of *`CADDFLAG`*, since it's only required for the *LUA* build.